### DISCUSSION TITLE
More options for tables

### DISCUSSION THREAD

**Discussion by DrZoidberg09** [chartbrew/chartbrew#117] [Ideas]
So far, I am super happy with Chartbrew using a lot of different charts and tables.

However, one thing would be nice: Being able to format the tables a little more comfortably apart for th SQL query.

This would be mostly:
- Selecting the name of the column
- Changing the order of the columns
- Formating the cells (e.g. truncate, date format, etc.)
- Width of the columns (rather nice to have)

What do you think?

**Comment by razvanilin** [comment]
Hey @DrZoidberg09, glad you're enjoying the experience so far!

Those are great suggestions for the tables. The naming and order have come up in my usage as well and really wanted to work on that. I think I wanted an extra push from users before implementing the functionality haha.

The width might be a bit of a tricky one to get right, but I can brainstorm ideas 👌

**Comment by razvanilin** [comment]
**Quick update**

The order of the columns can now be changed as of [v1.23.0](https://github.com/chartbrew/chartbrew/releases/tag/v1.23.0)

Here's a short video on how it works: https://www.loom.com/share/9a16d7efa43140818c81ab1db1f86fc1

**Comment by DrZoidberg09** [comment]
Thanks a lot!

That looks awesome. 

Also I have seen some more commits in this direction. Really looking forward to the next release.

**Reply by razvanilin** [reply]
Here's another update that is landing in the next version 😁 https://www.loom.com/share/658b82d38f07402fb304d97728c1fe65

**Comment by DrZoidberg09** [comment]
Looks super nice! Great to see the progress at Chartbrew. Keep up the good work!

**Comment by razvanilin** [comment]
More table updates!

It is now possible to format date and number columns ✨

Here's how it works as of now: https://www.loom.com/share/106bbd81809e4d1da9933d63e907437c
